apollo-router/src/plugins/telemetry/metrics/otlp.rs

```diff
@@ -41,8 +42,33 @@ impl MetricsConfigurator for super::super::otlp::Config {
             Some(exporter) => {
                 let exporter = opentelemetry_otlp::new_pipeline()
                     .metrics(
-                        opentelemetry::sdk::metrics::selectors::simple::inexpensive(),
-                        aggregation::stateless_temporality_selector(),
+                        selectors::simple::histogram([
+                            // 1-500 ms
```

Comment:
Just want to sanity check the units here. The fact that units are floats make me think they are in SI rather than millis.

Reply:
all the values the router generates are in seconds. I just changed and aligned those histograms to the ones used in prometheus

Reply:
there are conventions like https://opentelemetry.io/docs/reference/specification/metrics/semantic_conventions/http-metrics/#metric-httpclientduration

so far in the router we went with using the second as base unit, not the millisecond, so 1-500ms translates as 0.001..0.5

Reply:
That's good to know.